### PR TITLE
OCPBUGS-17497: Skip invariant load balancer test for None platform

### PR DESCRIPTION
**Pull request by pravin-dsilva**

### CODE REVIEW

**Comment by jaypoulz** [comment]
Hi @pravin-dsilva - can you retitle the PR so that it has:
OCPBUGS-17497: XXXXXXXXXXX

That way we can link it to the correct Jira

File: pkg/monitortests/network/disruptionserviceloadbalancer/monitortest.go [hunk 1]
```diff
@@ -101,7 +101,8 @@ func (w *availability) StartCollection(ctx context.Context, adminRESTConfig *res
 		infra.Status.PlatformStatus.Type == configv1.NutanixPlatformType ||
 		infra.Status.PlatformStatus.Type == configv1.VSpherePlatformType ||
 		infra.Status.PlatformStatus.Type == configv1.BareMetalPlatformType ||
-		infra.Status.PlatformStatus.Type == configv1.OpenStackPlatformType {
+		infra.Status.PlatformStatus.Type == configv1.OpenStackPlatformType ||
+		infra.Status.PlatformStatus.Type == configv1.NonePlatformType {
```
Comment (bparees) [inline pkg/monitortests/network/disruptionserviceloadbalancer/monitortest.go:105]: please add a comment in the code and in the commit message explaining why these tests are skipped on this platform

Comment (pravin-dsilva) [reply pkg/monitortests/network/disruptionserviceloadbalancer/monitortest.go:106]: Added comment.

**Comment by bparees** [comment]
ty

/lgtm

**Comment by jaypoulz** [comment]
@pravin-dsilva I missed this last time I retested, but can you run `make update-gofmt` and repush?

**Pull request merged**